app/models/submission_report.rb

```diff
@@ -0,0 +1,13 @@
+class SubmissionReport < ApplicationRecord
+  belongs_to :submission, class_name: 'TimelineEvent'
+
+  validates :description, presence: true
```

Comment:
1. `description` is shown as a `Test Report` in the UI. So maybe this column name should just reflect that?
2. `description` is currently a required field, meaning that a report is required even if automated tests are still in progress - this doesn't make sense. A test report would only be generated by once tests have completed. This should be an optional column, I think.